src/app.ts

```diff
@@ -17,4 +17,10 @@ app.use(express.urlencoded({ extended: false }));
 
 app.use('/', router);
 
+// tu zmienic nizej, p
 
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/DB')
```

Comment:
pass newUrlParser to remove the warning:
`mongoose.connect('mongodb://localhost:27017/DB', { useNewUrlParser: true })`